gateway/testdata/configuration.yaml

```
@@ -1,6 +1,14 @@
 server:
   address: localhost
   port: 8080
+  http_server_read_timeout: 20
```

```suggestion
  http_server_read_timeout: 20s
```

all the timeouts should be all human readable. A number is not enough. Also `1m` should be supported.

Use [time.ParseDuration](https://pkg.go.dev/time#ParseDuration)